### PR TITLE
Fixed an issue with triple click in nested editable.

### PR DESCRIPTION
**Pull request by pomek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Fixed a weird selection behavior on at least triple click. Triple click inside the nested editable will select the entire element now. Closes ckeditor/ckeditor5#1463.

---

Requires: https://github.com/ckeditor/ckeditor5-utils/pull/278.

### CODE REVIEW

File: tests/widget-integration.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+				} );
+
+				editor.conversion.for( 'downcast' )
+					.elementToElement( { model: 'paragraph', view: 'p' } )
+					.elementToElement( { model: 'inline', view: 'figure' } )
+					.elementToElement( { model: 'image', view: 'img' } )
+					.elementToElement( {
+						model: 'widget',
+						view: ( modelItem, viewWriter ) => {
+							const b = viewWriter.createAttributeElement( 'b' );
```
Comment (Reinmar) [inline tests/widget-integration.js]: Why do we need this `<b>` element here?

Comment (pomek) [reply tests/widget-integration.js]: We don't need them. I'll remove it.

**Comment by oleq** [comment]
I just thought, shouldn't we narrow this fix to Safari only?

**Comment by pomek** [comment]
Now CI will fail because we need to merge https://github.com/ckeditor/ckeditor5-utils/pull/278 first.

**Comment by oleq** [comment]
There's still an issue when the selection is at the end of the nested editable and double-clicked:

Kapture 2019-03-28 at 12 05 59

**Comment by pomek** [comment]
Even after fixing TC mentioned by @oleq, I am still able to reproduce the issue. Double click at the end of the line right before the widget:

image

Then double click:

image

Weird…

**Comment by oleq** [comment]
Extracted to https://github.com/ckeditor/ckeditor5-widget/issues/80.

**Pull request merged**